### DISCUSSION TITLE
failed to install neocmake and ruff

### DISCUSSION THREAD

**Discussion by 28arnab** [LazyVim/LazyVim#4198] [Q&A]
![image](https://github.com/user-attachments/assets/c33ed588-be94-474f-9d55-0098ba6cc148)

**Comment by dpetka2001** [comment]
The error tells you to check the Mason log to see more details. Do that.

**Comment by R1ky68** [comment]
In my case, I had the same issue just for the installation of **_ruff_** on my Pop!_OS 22.04 LTS.

The error logs are not that useful, because typing _**:MasonLog**_  returns the following line:
`[ERROR Wed 29 Jan 2025 03:58:46 PM CET] ...e/nvim/lazy/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=sqlfluff) error=spawn: python3 failed with exit code 1 and signal 0.`

So I thought that there was a problem with the package `sqlfluff` and maybe it wasn't finding the source, but this was not the problem.

I solved the issue by following [this](https://github.com/williamboman/mason.nvim/discussions/1823#discussioncomment-11510462) answer.

Hope this helps someone!
